appstore/validator.go

```diff
@@ -115,6 +115,7 @@ func (c *Client) Verify(ctx context.Context, reqBody IAPRequest, result interfac
 }
 
 // VerifyWithStatus sends receipts and gets validation result with status code
+// If apple verify receipt server unhealthy and response http status code 5xx, this status code will return
```

Comment:
Could you please correct the comment to `If the Apple verification receipt server is unhealthy and responds with an HTTP status code in the 5xx range, that status code will be returned.` ?

Reply:
already change the comment.